manifests/j/JAMSoftware/TreeSize/9.1/JAMSoftware.TreeSize.installer.yaml

```diff
@@ -0,0 +1,25 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: JAMSoftware.TreeSize
+PackageVersion: 9.1
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: TreeSize_is1
+AppsAndFeaturesEntries:
+- DisplayName: TreeSize V9.0.3 (64 bit)

```

Comment:
```suggestion
```
Since you are automating these PRs, I would suggest removing this bit as it would need to be manually edited for each new release. E.g., for this PR, DIsplayName is `TreeSize V9.1`. DisplayName is mostly intended for good package correlation, but I think we have that covered already with Publisher and ProductCode values being a 1-1 map with the registry values (and PackageName will also probably be matched with the fuzzy heuristics)

Reply:
Yes, that seems to make sense .. even though we had some trouble with the fuzzy heuristics in the past (our packages changed since then though, so I assume they should be resolved already)